maintainers/maintainer-list.nix

```diff
@@ -2648,6 +2648,11 @@
     github = "limeytexan";
     name = "Michael Brantley";
   };
+  linarcx = {
```

Comment:
Consider splitting this out into a separate commit, but keeping it in this PR, since it's needed.